dev-tools/reconfigurator-cli/tests/output/cmds-set-mgs-updates-stdout

```diff
@@ -422,6 +425,9 @@ parent:    ad97e762-7bf1-45a6-a98f-60afb7e491c0
     sled      2      model2        serial2         e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855   1.1.0              Sp { expected_active_version: ArtifactVersion("1.0.0"), expected_inactive_version: Version(ArtifactVersion("1.0.1")) }
 
 
+Nothing to report on planning for blueprint cca24b71-09b5-4042-9185-b33e9f2ebba0.
```

Comment:
What does this message mean?  Does it mean the blueprint is the same as the parent?

edit: I think it might also mean "we loaded this from the database and so have no report for it".  Maybe this should say "Empty planning report ..."?  That might avoid confusing someone who sees this and thinks there have been no changes, when it might just be that we loaded this from the database.